### PR TITLE
Add confirmation to app uninstall

### PR DESCRIPTION
**Pull request by TheMoonThatRises**
Prompts user if they want to uninstall the app #261 (sorry for repeat, had wrong branch name)
UninstallConfirmPopup

### CODE REVIEW

File: PlayCover/Views/App Views/PlayAppView.swift [hunk 1]
```diff
@@ -106,7 +107,7 @@ struct PlayAppView: View {
                     Text("playapp.clearPreferences")
                 })
                 Button(action: {
-                    app.deleteApp()
+                    showDeleteConfirmation = true
```
Comment (IsaacMarovitz) [inline PlayCover/Views/App Views/PlayAppView.swift]: `showDeleteConfirmation.toggle()` would be better here

**Comment by amirsaam** [comment]
Glad to see. Thanks!

File: PlayCover/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -6,6 +6,8 @@
 "playapp.exportKm" = "Export Keymapping";
 "playapp.exportKmPanel.fieldLabel" = "PlayMap Name:";
 "playapp.delete" = "Uninstall App";
+"playapp.deleteConfirm" = "Yes, Uninstall";
```
Comment (IsaacMarovitz) [inline PlayCover/en.lproj/Localizable.strings]: Change this button to just "Uninstall" or "Confirm" or something like that

**Comment by IsaacMarovitz** [comment]
#312 Should be merged before merging this PR

**Review by Depal1 — APPROVED** [review]

**Pull request merged**